packages/ez_search/ez_search.0.2.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+synopsis: "The ez_search project"
+description: """\
+This is the description
+of the ez_search OCaml project
+"""
+authors: ["Fabrice Le Fessant <fabrice.le_fessant@origin-labs.com>"]
+maintainer: ["Fabrice Le Fessant <fabrice.le_fessant@origin-labs.com>"]
+homepage: "https://ocamlpro.github.io/ez_search"
+doc: "https://ocamlpro.github.io/ez_search/sphinx"
+bug-reports: "https://github.com/ocamlpro/ez_search/issues"
+dev-repo: "git+https://github.com/ocamlpro/ez_search.git"
+tags: "org:ocamlpro"
+build: [
+  ["dune" "subst"] {dev}
+  ["sh" "-c" "./scripts/before.sh build '%{name}%'"]
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+  ["sh" "-c" "./scripts/after.sh build '%{name}%'"]
+]
+install: [
+  ["sh" "-c" "./scripts/before.sh install '%{name}%'"]
+]
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {>= "2.7.0"}
+  "re" {>= "1.8.0" & < "2.0.0"}
+  "parmap" {>= "1.2.3" & < "2.0.0"}
+  "ez_file" {>= "0.2"}
+  "ez_cmdliner" {>= "0.2"}
+  "ppx_inline_test" {with-test}
+  "ppx_expect" {with-test}
+  "odoc" {with-doc}
+  "ocamlformat" {with-test}
+]
```

Comment:
```suggestion
  "conf-libpcre"
]
```
it looks like you might want to require libpcre to be installed:
```
#=== ERROR while compiling ez_search.0.2.0 ====================================#
# context              2.0.10 | linux/x86_64 | ocaml-base-compiler.4.07.1 | pinned(https://github.com/ocamlpro/ez_search/archive/v0.2.0.tar.gz)
# path                 ~/.opam/4.07/.opam-switch/build/ez_search.0.2.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p ez_search -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/ez_search-4017-6d07ae.env
# output-file          ~/.opam/log/ez_search-4017-6d07ae.out
### output ###
#        ocaml src/ez_search/version.ml
# fatal: not a git repository (or any parent up to mount point /)
# Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
# fatal: not a git repository (or any parent up to mount point /)
# Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
#   ocamlmklib src/ez_search/dllez_search_stubs.so,src/ez_search/libez_search_stubs.a (exit 2)
# (cd _build/default && /home/opam/.opam/4.07/bin/ocamlmklib.opt -g -o src/ez_search/ez_search_stubs src/ez_search/mapfile_stubs.o src/ez_search/pcre_stubs.o src/ez_search/strfind.o src/ez_search/strstubs.o -lpcre)
# /usr/bin/ld: cannot find -lpcre
# collect2: error: ld returned 1 exit status
```

Reply:
In fact no, `libpcre` is no longer needed, we jut forgot to re-tag the project, all pcre related dependencies are are deprecated.
Hence the need for a new tag